proto-google-cloud-spanner-host/src/main/proto/google/spanner/host/login.proto

```
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package spanner.experimental;
```

Buf (1.54.0)

Files with package "spanner.experimental" must be within a directory "spanner/experimental" relative to root but were in directory "proto-google-cloud-spanner-host/src/main/proto/google/spanner/host".

(PACKAGE_DIRECTORY_MATCH)